src/libraries/System.Collections.Immutable/tests/ImmutableHashSetBuilderTest.cs

```diff
@@ -332,5 +332,31 @@ public void ToImmutableHashSet()
             ImmutableHashSet<int>.Builder nullBuilder = null;
             AssertExtensions.Throws<ArgumentNullException>("builder", () => nullBuilder.ToImmutableHashSet());
         }
+
+        [Fact]
+        public void TryGetValue()
+        {
+            var builder = ImmutableHashSet.Create(1, 2, 3).ToBuilder();
+            Assert.True(builder.TryGetValue(2, out _));
+
+            builder = ImmutableHashSet.Create(CustomEqualityComparer.Instance, 1, 2, 3, 4).ToBuilder();
+            var existing = 0;
+            Assert.True(builder.TryGetValue(5, out existing));
+            Assert.NotEqual(5, existing);
```

Comment:
This NotEqual check isn't necessary.  An int can't be both 4 and 5, so the next Assert.Equal(4, existing) covers it.